src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -197,6 +207,30 @@ define([
             }
         },
 
+        _executeNanoFlow: function() {
+            logger.debug(this.id + "._executeNanoFlow");
+            if (!this._contextObj) {
+                return;
+            }
+
+            if (this.nanoflow.nanoflow) {
+                mx.data.callNanoflow({
+                    nanoflow: this.nanoflow,
+                    origin: this.mxform,
+                    context: this.mxcontext,
+                    callback: lang.hitch(this, function(result) {
+                        if (!result) {
+                            logger.debug(this.id + "._executeNanoFlow callback, stopping timer");
```

Comment:
Give a proper error message using `mx.ui.error` change message to `("Error while executing the nanoflow, stopping timer")`